client/src/plus/updates/index.tsx

```diff
@@ -34,13 +37,97 @@ const CATEGORY_TO_NAME = {
   webextensions: "Web Extensions",
 };
 
+const BROWSERS = {
```

Comment:
We should one day look to put this context in the API

Reply:
The alternative would be to get it from `@mdn/browser-compat-data`, to which we (should) have access during the build.

Reply:
Resolved via 6019f8f71379f9bfe1ff7d943600a9de667c9284.